model/flow/chunk.go

```diff
@@ -66,6 +70,17 @@ func (cl ChunkList) Items() []*Chunk {
 	return cl
 }
 
+func (cl ChunkList) Indices() []uint64 {
+	indices := make([]uint64, len(cl))
+	for i, chunk := range cl {
+		indices[i] = chunk.Index
+	}
+
+	sortkeys.Uint64s(indices)
```

Comment:
I'm not sure if we need to sort. 
```suggestion

```

Reply:
@zhangchiqing I ran the local net and it is unsorted.

Reply:
It's seems we only use the indices for logging. Do we care if the indices is sorted when we are logging it?

In the tests, we could sort it after it's returned